core/config/config.js

```diff
@@ -1,5 +1,5 @@
-export const MODE_DEBUG                                        = true;
```

Comment:
The changes on this file are not related to the pull request, so they should not be included in the pr.